salt/systemd/dqt_celery.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
```

Comment:
Like other comment, should this be called pwfy_dqt_celery.service?

Reply:
OK